### DISCUSSION TITLE
i have small problem

### DISCUSSION THREAD

**Discussion by MohamedEbrahimSaad** [serenity-is/Serenity#5311] [Help]
when i upload images of the entity every thing work perfect but after 24 hours i think the image being deleted and i should upload it again is there any thing i should do 

also i want to know about temporary file how can i change the saved images from it to another folder or even how to make serenity not removing files in this folder

**Comment by MohamedEbrahimSaad** [comment]
i found that when imageuploadeditor is set on the xxxForm.cs it becomes in the temporary folder and when we set it in the xxxRow.cs it becomes in the basic folder if i'm right please close the issue

**Comment by VictorTomaili** [comment]
Please open questions on discussion tab.

**Reply by MohamedEbrahimSaad** [reply]
I'm sorry , and thanks for mentioning this point for me 💙
